### PR TITLE
Adding session02 code

### PR DESCRIPTION
**Pull request by ervbrian**
Added GridPrinter.py for displaying ascii grid patterns. 
Added series.py for calculating Fibonacci, Lucas and custom summation values.

### CODE REVIEW

**Comment by nairraghav** [comment]
Great job Brian! It looks like you're missing the initial grid printer function, otherwise the rest looks great.

File: students/BrianE/session02/GridPrinter.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+"""
+GridPrinter.py
+
+Prints ascii grid based on desired box size and grid size.
+"""
+
+
```
Comment (nairraghav) [inline students/BrianE/session02/GridPrinter.py:7]: Don't forget to implement the initial grid which only takes 1 parameter.

File: students/BrianE/session02/series.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    assert lucas(6) == 18
+    assert lucas(7) == 29
+
+    # test if sum_series matched fibonacci
+    assert sum_series(5) == fibonacci(5)
+
+    # test if sum_series matched lucas
+    assert sum_series(5, 2, 1) == lucas(5)
+    print("All assertions pass")
+
```
Comment (nairraghav) [inline students/BrianE/session02/series.py:94]: Just a side note, you may want to add some test cases to validate that the custom series works: assert sum_series(5, 5, 7) == 50

Comment (ervbrian) [reply students/BrianE/session02/series.py:94]: Added a few more assertions.

File: students/BrianE/session02/GridPrinter.py (hunk 1 above)
Comment (ervbrian) [reply students/BrianE/session02/GridPrinter.py:7]: I've added an example of how my function supports a single parameter (box_size). The default value of 2 for grid_size supports calling grid(5). 

Let me know if you want a completely separate function with only a single parameter defined and I'll add that, instead.

Comment (ervbrian) [reply students/BrianE/session02/GridPrinter.py:7]: Went ahead and added the initial print_grid() function just in case.

**Comment by nairraghav** [comment]
Nice job Brian, this looks great!

**Pull request merged**